Sources/Mockable/Builder/FunctionBuilders/FunctionReturnBuilder.swift

```diff
@@ -50,3 +50,10 @@ public struct FunctionReturnBuilder<T: Mockable, ParentBuilder: EffectBuilder<T>
         return .init(mocker: mocker)
     }
 }
+
+extension FunctionReturnBuilder where ReturnType == () {
+    @discardableResult
+    public func justRuns() -> ParentBuilder {
```

Comment:
Void functions are already not required to be mocked explicitly.

You can call any void function of a `MockService` without a given clause and it will return.